src/views/Home/components/DaysLeft/index.jsx

```diff
@@ -69,6 +75,9 @@ const DaysLeft = () => {
                 {daysFinished}
               </Typography>
               <Typography className={styles.label}>Days Finished</Typography>
+              <Typography variant="body2" className={styles.label2}>
+                {`Current Term: ${first} - ${last}`}
```

Comment:
In JSX, you can put javascript value into HTML more easily:
```suggestion
                Current Term: {first} - {last}
```

Reply:
Thank you for the help, and I changed the small refactoring.